app/assets/javascripts/transactions.js

```diff
@@ -19,7 +19,7 @@ $(document).ready(function () {
     append('<input type="hidden" name="service_feedback[javascript_enabled]" value="true"/>').
     append($('<input type="hidden" name="referrer">').val(document.referrer || "unknown"));
 
-  $('#completed-transaction-form button.button').click(function() {
+  $('#completed-transaction-form button[type="submit"]').click(function() {
```

Comment:
Is it worth changing this selector for a JavaScript-hook-named class? The class is named to be JS only, and functionality can survive other attribute changes.

```suggestion
  $('.js-completed-transaction-form-button').click(function() {
```

Reply:
Agreed, except the element in question is a component. I chose `button[type="submit"]` because it most clearly shows which button the JS acts on and will survive changes as you said. But yeah, there's nothing in the template to indicate that there's JS involved (to be fair, there wasn't before, I only found this by accident).

Reply:
Ah boo. However, I do wonder if we use enough JavaScript for this to be a useful thing?